config/common/peer-vpn-iatlasvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/peer-route-config.yaml
+stack_name: peer-vpn-iatlasvpc
+dependencies:
+  - common/iatlasvpc.yaml
+parameters:
+  PeeringConnectionId: pcx-0f97213011da78981
+  VpcPrivateRouteTable: rtb-03749e8c822352561
```

Comment:
the VpcPrivateRouteTable for iatlasvpc is `rtb-073c34d41ecd890b1` 

please double check these values